src/lib/containers/query_wrapper_plot_nav/TopLevelControls.tsx

```diff
@@ -167,8 +174,9 @@ const TopLevelControls = (
           {controls.map(control => {
             const { key, icon, tooltipText } = control
             if (
-              key === 'showDownloadConfirmation' &&
-              (!isFileView || hideDownload)
```

Comment:
I think this was a bug introduced when implementing the Download Cart.  Download options are available for other Tables too (e.g. Export Table), you just can't add the query results to the Download Cart.  This case is already handled by the DownloadOptions component.